src/Swashbuckle.AspNetCore.Swagger/SwaggerOptions.cs

```diff
@@ -25,9 +25,15 @@ public SwaggerOptions()
         /// </summary>
         public bool SerializeAsV2 { get; set; }
 
+        /// <summary>
+        /// Provide a custom ISwaggerDocumentSerializer implementation, to have more control over how the swagger document is exactly serialized.
+        /// </summary>
+        /// <remarks>For the CLI tool to be able to use this, this needs to be set during ConfigureServices.</remarks>
+        public ISwaggerDocumentSerializer CustomDocumentSerializer { get; set; } = null;
```

Comment:
Tweaked the wording here, as there are other ways to configure an ASP.NET Core application's dependencies other than using `ConfigureServices()`.

```suggestion
        /// <summary>
        /// Gets or sets an optional custom <see cref="ISwaggerDocumentSerializer"/> implementation to use to serialize Swagger documents.
        /// </summary>
        /// <remarks>For the CLI tool to be able to use this, this needs to be configured for use in the service collection of your application.</remarks>
        public ISwaggerDocumentSerializer CustomDocumentSerializer { get; set; }
```